docs/installation.rst

```diff
@@ -402,6 +402,9 @@ Here's a list of some of the recommended packages.
 | Vertica          | ``pip install                         |  ``vertica+vertica_python://``                  |
 |                  | sqlalchemy-vertica-python``           |                                                 |
 +------------------+---------------------------------------+-------------------------------------------------+
+| Hana             | ``pip install hdbcli``                |  ``hana://``                  |
```

Comment:
based on other suggested changes using setuptools' `extras_require`, this should be `pip install superset[hana]`